src/net.h

```diff
@@ -874,6 +875,25 @@ class CNode
         assert(false);
     }
 
+    std::string ConnectionTypeToString() const {
+        switch (m_conn_type) {
+            case ConnectionType::INBOUND:
+                return "inbound";
+            case ConnectionType::MANUAL:
+                return "manual";
+            case ConnectionType::FEELER:
+                return "feeler";
+            case ConnectionType::OUTBOUND:
+                return "outbound full relay";
+            case ConnectionType::BLOCK_RELAY:
+                return "block relay only";
```

Comment:
nit: Would it make sense to replace the spaces with `-` to preserve the previous format of the log message? Also, if this was used in an arbitrary sentence, it might not be clear that this is a single word/"operator"

Reply:
done